front/components/assistant_builder/BuilderLayout.tsx

```diff
@@ -12,12 +12,16 @@ interface BuilderLayoutProps {
 export function BuilderLayout({ leftPanel, rightPanel }: BuilderLayoutProps) {
   return (
     <div className="flex h-full w-full">
-      <ResizablePanelGroup direction="horizontal" className="h-full w-full">
-        <ResizablePanel defaultSize={50} minSize={30}>
+      <ResizablePanelGroup
+        autoSaveId="assistant-builder-layout"
```

Comment:
Didn't have to implement anything it's already supported by [the library](https://github.com/bvaughn/react-resizable-panels/tree/main?tab=readme-ov-file#can-panel-sizes-be-persistent) 😄